core/src/main/scala/kafka/coordinator/GroupMetadataManager.scala

```diff
@@ -659,13 +682,20 @@ class GroupMetadataManager(val brokerId: Int,
       config.offsetsTopicNumPartitions
   }
 
-  private def getMessageFormatVersionAndTimestamp(partition: Int): (Byte, Long) = {
+  /**
+   * Check if the replica is local and return the message format version and timestamp
+   *
+   * @param partition   Partition of GroupMetadataTopic
+   *
+   * @return            Option[(MessageFormatVersion, TimeStamp)] if replica is local, None otherwise
```

Comment:
I think you mean `Some[(MessageFormatVersion, Timestamp)]...`. Nit: do we need a new line between param and return?